descqa/CorrelationsTwoPoint.py

```diff
@@ -237,6 +298,70 @@ def score_and_test(corr_data): # pylint: disable=unused-argument
         return TestResult(inspect_only=True)
 
 
+    def get_jackknife_randoms(self, N_jack, catalog_data, ra='ra', dec='dec'):
+        """
+        Computes the jackknife regions and random catalogs for each region 
+        Parameters
+        ----------
+        N_jack : number of regions
+        catalog_data : input catalog
+
+        Returns
+        -------
+        jack_labels: array of regions in catalog data
+        randoms: dict of randoms labeled by region
+        """
+        catalogs = {}
+        #cluster
+        nn = np.stack((catalog_data[ra], catalog_data[dec]), axis=1)
+        _, jack_labels, _ = k_means(n_clusters=N_jack, random_state=0, X=nn)
+
+        randoms = {}
+        for nj in range(N_jack):
+            catalog_data_jk = dict(zip(catalog_data.keys(), [v[(jack_labels != nj)] for v in catalog_data.values()]))
+            rand_cat, rr = self.generate_processed_randoms(catalog_data_jk) #get randoms for this footprint
+            #print('nj = {}, area = {:.2f}'.format(nj, self.check_footprint(catalog_data_jk))) #check footprint
```

Comment:
commented out print statement, perhaps this should be deleted?

Reply:
Done